src/context/UserContext.js

```diff
@@ -1 +1,15 @@
-// TODO -- create the user provider!
+import { createContext, useState } from 'react';
+import { getUser } from '../services/auth';
+
+const UserContext = createContext();
+const UserProvider = ({ children }) => {
+  const [user, setUser] = useState(getUser);
```

Comment:
You'll want to invoke the `getUser` function here so that the value it returns is then passed to `useState()`. As it is right now this is setting the value of user to the entire `getUser` function.

```js
const [user, setUser] = useState(getUser());